ianmccurry11/TODO_List
# Added password confirmation to registration

# Feature Overview
## Implemented check for password verification when registering account
With the addition of another password input to let users write their password and double check that it is the same in both fields, the inputs will be removed if this is not the case.

## Testing (how to view feature)
1. Go to register page
2. create new username and password
3. put a different string from password in the confirm-password box
4. hit submit
5. you will see the fields will become empty
